webapp/client/src/components/Details.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Details from "./Details";
+
+describe("Details with content paragraphs", () => {
+  beforeEach(() => {
+    render(
+      <Details title="title" detailsId="my-id">
+        {{
+          paragraphs: ["content paragraph"],
+        }}
+      </Details>
+    );
+  });
+
+  it("should render the title", () => {
+    expect(screen.getByText("title")).toBeInTheDocument();
+  });
+
+  it("should render paragraphs", () => {
+    expect(screen.getByText("content paragraph").nodeName).toEqual("P");
+  });
```

Comment:
Correct. There's also [`toBeVisible`](https://github.com/testing-library/jest-dom#tobevisible) but testing the opening behaviour is one of the things I didn't get to work but also didn't want to spend too much time on.

I'm assuming it's because React renders the component and Bootstrap only hides it (with jQuery) afterwards and maybe there's a timing issue with the tests?